dll/kernel32.cpp

```diff
@@ -624,17 +624,15 @@ namespace kernel32 {
 		const auto lpFileNameA = wideStringToString(lpFileName);
 		std::filesystem::path absPath = std::filesystem::absolute(files::pathFromWindows(lpFileNameA.c_str()));
 		std::string absStr = files::pathToWindows(absPath);
-		const auto absStrW = stringToWideString(absStr.c_str());
+		const auto absStrW = stringToWideString(absStr.c_str()).data();
```

Comment:
The vector will be deallocated at the end of the expression, I believe. Call data() at the use sites instead